recipes/pyglpk/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pyglpk" %}
+{% set version = "0.4.6" %}
+{% set source = "glpk" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ source[0] }}/{{ source }}/{{ source }}-{{ version }}.tar.gz
+  sha256: 8739fcf6513c2f225ac525c2a43c89f10de27b546983e7900235ea87c85a1b74
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - glpk
+    - setuptools >=38.6.0
+    - setuptools_scm
+  run:
+    - python
+    - glpk
+
+test:
+  imports:
+    - glpk
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: http://tfinley.net/software/pyglpk/
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE.txt
```

Comment:
Please change this to `LICENSE` to match the name of the license file in the source distribution.